extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -283,6 +283,15 @@ export function validateRtcConfig_(element) {
     return null;
   }
 
+  const vendors = rtcConfig['vendors'] || {};
+  Object.keys(vendors).forEach(vendor => {
+    Object.keys(vendors[vendor]).forEach(key => {
+      if (isObject(vendors[vendor][key])) {
+        vendors[vendor][key] = JSON.stringify(vendors[vendor][key]);
```

Comment:
Maybe a comment explaining why this is being done?

Reply:
Will do